src/components/index/intro.tsx

```diff
@@ -1,38 +1,36 @@
 import { IntroAnimation } from "./intro-splash";
 import { Button } from "../shared/button";
 import "./styles/intro.css";
+import type { Introduction, IntroSplashEntry } from "../../utils/api";
 
 interface IntroProps {
+  intro_data: Introduction;
+  intro_splash_data: IntroSplashEntry[];
   getNetLogoSection: React.RefObject<HTMLDivElement>;
 }
 
-const Intro: React.FC<IntroProps> = ({ getNetLogoSection }) => {
+const Intro: React.FC<IntroProps> = ({
```

Comment:
Why do you use `React.FC` in some places and function declaration syntax in others (like for `Body` in body.tsx)? Not being very knowledgable about React and Typescript, I don't know what the advantages are to each. If they are equivalent, I would like it be consistent across the code base. If not, please enlighten me as to the advantages of each.